Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use an official Python runtime as a parent image
+FROM python:3.10-slim
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Install SSL certificates, update certifi, install pipx and move to path
+RUN apt-get update && apt-get install -y ca-certificates git curl make \
+    && pip install --upgrade certifi \
+    && pip install pipx \
+    && pipx ensurepath
+
+# Install goose-ai CLI using pipx
+RUN pipx install goose-ai
```

Comment:
If the image was built before and this layer is cached, the user won't have the latest `goose-ai` if there is a new release. 

We could either to disable the cache of this layer  or the user can install the latest in the docker container

Reply:
@lifeizhou-ap what about if this instead was built as part of the py release workflow - and pushed as a versioned image to the GH docker registry - would that be suitable?